Jenkinsfile-nightly

```diff
@@ -1,8 +1,10 @@
-OLTP_TERMINALS = ['1','2','4','8','16']
-
 pipeline {
     agent none
 
+    environment{
+        PSS_USERNAME= credentials('pss-username')
+        PSS_PASSWORD= credentials('pss-password')
```

Comment:
We need to make sure Chad adds these right? or did he already? We technically can just use one username-password type credential entry similar to http://jenkins.db.cs.cmu.edu:8080/job/testing-team/credentials/store/folder/domain/_/credential/pss-creator-staging/

Reply:
I think the one we created can be used globally.